test/unit/gateways/commerce_hub_test.rb

```diff
@@ -128,7 +128,7 @@ def test_failed_purchase_and_authorize
 
     response = @gateway.authorize(@amount, @credit_card, @options)
     assert_failure response
-    assert_equal 'HOST', response.error_code
+    assert_equal 'string', response.error_code
```

Comment:
just for curiosity, what exactly is being tested here? 
e.g. 
```
response.error_code == 'string' or response.error_code is a String 
```

Reply:
Ahhh nothing special, that's the value of the response code on the string that is being used as a successful response:

![tmux](https://user-images.githubusercontent.com/133580/220969762-83adcc00-b27f-4015-9bb1-5b99edb76ab0.png)